README.md

```diff
@@ -58,6 +58,15 @@ If you would like to run Sara on your website, follow the instructions
 [here](https://github.com/mrbot-ai/rasa-webchat) to place the chat widget on
 your website.
 
+## To test Sara:
+
+After doing a `rasa train`, run the command:
+
+```bash
+rasa test nlu -u test/test_data.json --model models/$(ls models)
```

Comment:
this will only work if you only have one model right? it automatically takes the most recent, so i think this will work:
```suggestion
rasa test nlu -u test/test_data.json --model models
```